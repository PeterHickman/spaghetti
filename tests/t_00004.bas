10 REM Checking division
20 LET A = 3 / 2
30 PRINT A
40 LET A = 2 / 3
50 PRINT A
60 LET A = 4 / -1
70 PRINT A
80 LET A = -1 / 4
90 PRINT A
