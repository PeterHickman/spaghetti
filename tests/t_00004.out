1.5
0.6666666666666666
-4
-0.25
